cclib/attribute_parsers/dispersionenergies.py

```diff
@@ -29,10 +29,25 @@ def psi4(file_handler, ccdata) -> Optional[dict]:
             this_dispersionenergies.append(dispersion)
             return {dispersionenergies.__name__: np.array(this_dispersionenergies)}
         return None
-
         # The geometry convergence targets and values are printed in a table, with the legends
 
-    known_codes = ["psi4"]
+    @staticmethod
+    def gaussian(file_handler, ccdata) -> Optional[dict]:
+        ccsd_trigger = "* CCSD total energy"  # noqa: F841
+        ccsd_t_trigger = "* CCSD(T) total energy"  # noqa: F841
+        line = file_handler.last_line
+        if getattr(ccdata, "dispersionenergies") is None:
+            this_dispersionenergies = []
+        else:
+            this_dispersionenergies = ccdata.dispersionenergies
+
+        if "Dispersion energy=" in line:
+            dispersion = float(line.split()[-2])
+            this_dispersionenergies.append(dispersion)
+            return {dispersionenergies.__name__: np.array(this_dispersionenergies)}
+        return None
+
+    known_codes = ["psi4", "gaussian"]
```

Comment:
could we  also move this to the top of the class definition for consistency among attributes? This looks like my mistake initially, but just catching it now, my apologies again.

Reply:
done!